.github/workflows/ClangFormat.yml

```diff
@@ -15,6 +15,6 @@ jobs:
           install-llvm: true # Needed to configure jlm
           install-clang-format: true
       - name: "Configure jlm with HLS and MLIR enabled"
-        run: ./configure.sh --enable-mlir --enable-hls
+        run: ./configure.sh --enable-mlir /usr --enable-hls /usr
```

Comment:
The two extra flags sounds like the really clean solution.

Reply:
@phate @sjalander we could also do
```
--enable-mlir=/custom/path
--enable-mlir #uses default path ./usr
```
This might go against the rules of `getopt` somewhat, so some people might expect `--a=x` to be identical to `--a x`, but I know that `clang` for instance breaks this rule with `-flto=thin` and `-flto thin` not meaning the same thing at all.

As this makes it very explicit when you specify a custom path, we do not need to perform a check if the folder exists. If you try building with an installation folder that does not exist, you will quickly get errors

Reply:
@haved Fine with me.